modules/heat_conduction/tests/verify_against_analytical/2d_steady_state_final_prob.i

```diff
@@ -0,0 +1,119 @@
+# This test solves a 2D steady state heat equation
+# The error is found by comparing to the analytical solution
+
+# Note that the thermal conductivity, specific heat, and density in this problem
+# Are set to 1, and need to be changed to the constants of the material being
+# Analyzed
```

Comment:
Since you have an analytical solution for this problem, I'd like you to go ahead and describe the problem setup and show the solution here in the test file header. If you have a reference to where we can quickly find the solution that would be very valuable as well. If you want to take a look at some well documented tests. Look at the `.i` files in this directory: https://github.com/idaholab/moose/tree/devel/modules/combined/tests/gap_heat_transfer_htonly